docs/resources.md

```diff
@@ -6,8 +6,61 @@
 ## JavaDocs
 
 - [WPILib JavaDocs](https://first.wpi.edu/FRC/roborio/release/docs/java/index.html)
+- [Lib199 JavaDocs](https://deepbluerobotics.github.io/lib199/)
 - [CTRE JavaDocs (TalonSRX, VictorSPX)](http://www.ctr-electronics.com/downloads/api/java/html/index.html)
 - [REV Robotics JavaDocs (SPARK)](http://www.revrobotics.com/content/sw/max/sw-docs/java/index.html)
 - [Kauai Labs (AHRS)](https://www.kauailabs.com/public_files/navx-mxp/apidocs/java/index.html)
 - [Playing With Fusion (TimeOfFlight)](https://www.playingwithfusion.com/frc/2020/javadoc/index.html)
 - [JUnit 4 JavaDocs](https://junit.org/junit4/javadoc/latest/index.html)
+
+## Obscurities
+
+- [Updating Lib199 Video](https://drive.google.com/file/d/1CLS0XMqJ6Y5_mLtYdS3HUR-l4us7WAk-/view)
+- [Code Styling Guide](https://docs.google.com/document/d/1RVMY4_c5qCdh_hqiG-O33_-PpMKZtKSpayKp2bSBjTo/edit)
+- [Complex Obscure Buxfixes Doc](https://docs.google.com/document/d/1PtT4ICuIJJ_EW_JgU_YRrFyJk2P9CibrkjtXtpVGjXE/edit)
+
+### Github Usernames
+```
+Ana Hoffman -------------- VoidShark 
+Arianna Hsu -------------- AriannaHsu 
+David Elliston ----------- delliston 
+Jack Hansen -------------- 0sani 
+John Hurley -------------- hohnjurley 
```

Comment:
Sensitive info like this should not go public on our website. Instead, have a link to the doc on the website and make sure the google doc is only shared to people in the team.

Reply:
^^^ Still unresolved. Also don't leave the driver station serial numbers public no matter what.